jihyeon/app.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const cors = require('cors');
+const morgan = require ("morgan");
+const http = require ("http");
+require("dotenv").config();
```

Comment:
```suggestion
require("dotenv").config();

const express = require('express');
const cors = require('cors');
const morgan = require ("morgan");
```
1. 서버가 실행될 때, 초기에 환경변수를 읽어오는 코드는 보통 상단에 작성됩니다.
2. http 모듈은 node.js의 built-in 모듈로서, 인터넷을 통해 들어오는 네트워크 요청을 코드로서 처리할 수 있도록 할 수 있게 해주는 모듈입니다. 우리는 http의 기능에 더불어 보다 더 많은 기능을, 빠르고 쉽게 제공해주는 Express를 활용하기 때문에 http 모듈은 현재 필요하지 않습니다.